lib/global-styles-and-settings.php

```diff
@@ -168,6 +168,82 @@ function gutenberg_get_global_styles_custom_css() {
 	return $stylesheet;
 }
 
+/**
+ * Gets the global styles base custom CSS from theme.json.
+ *
+ * @since 6.6.0
+ *
+ * @return string The global base custom CSS.
+ */
+function gutenberg_get_global_styles_base_custom_css() {
```

Comment:
Can we simplify further and remove this function and `gutenberg_get_global_styles_custom_css` in favor of `gutenberg_get_global_stylesheet`? The `styles.custom` property is still the only one that has a dedicated function.